test/test_alexa_app_customslot.js

```diff
@@ -12,7 +12,8 @@ import * as Alexa from '..';
 describe("Alexa", function() {
 
   context("#alexa.request.slot", function () {
-    var testReq;
+    var testReq = mockHelper.load("intent_request_airport_info.json");
```

Comment:
Any reason you assigned `testReq` in the context? the previous commit was perfect.